src/server/cluster/cluster_family.cc

```diff
@@ -764,29 +776,27 @@ std::shared_ptr<OutgoingMigration> ClusterFamily::CreateOutgoingMigration(std::s
 
 void ClusterFamily::DflyMigrateFlow(CmdArgList args, ConnectionContext* cntx) {
   CmdArgParser parser{args};
-  auto [sync_id, shard_id] = parser.Next<uint32_t, uint32_t>();
+  auto [port, shard_id] = parser.Next<uint32_t, uint32_t>();
 
   if (auto err = parser.Error(); err) {
     return cntx->SendError(err->MakeReply());
   }
 
-  VLOG(1) << "Create flow sync_id: " << sync_id << " shard_id: " << shard_id;
+  auto host_ip = cntx->conn()->RemoteEndpointAddress();
```

Comment:
I'm not an expert in networking, but in the general case the host you've received the message from isn't neccessary the host to send your request to... Your `home pc -> router -> outside world` with port forwarding as an example 🙂 
I _don't_ know how modern cloud networks look and whether there can be such cases

Reply:
If the node is behind the NAT we can not create a direct connection at all. In any case, it's a temporary code for further changes. I will remove this connection in the next PR